device/marvell/arm64-marvell_rd98DX35xx-r0/installer.conf

```diff
@@ -1 +1 @@
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="default_hugepagesz=32M hugepages=4"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="default_hugepagesz=32M hugepages=4 mem=4G"
```

Comment:
why do we need this mem=4G? what is the purpose for this?

Reply:
HWSKU represented by this platform can be loaded on 4G or 8G RAM boards. This change is restricting the memory to 4G, as we have some open kernel issues due to some hardware component having only 32-bit addressing. Once we have solution to them, we plan to remove this restriction.